src/main/java/frc/robot/Constants.java

```diff
@@ -81,7 +81,14 @@
 
         /********** Tuning Constants **********/
         public static final double CONVEYOR_POWER = .2;
-        public static final double GATE_WHEEL_POWER = .75;
+        public static final double GATE_WHEEL_SHOOTING_POWER = .75;
+        public static final double GATE_WHEEL_INDEXING_POWER = .4;
+
+        public static final double GATE_WHEEL_INDEXING_DEGREES = 360; // Rotate the gate by this much to index a cargo
+        public static final double GATE_WHEEL_CURRENT_THRESHOLD = 20; // Amps
+
+        /******** Physical Constants ********/
+        public static final double GATE_WHEEL_GEAR_RATIO = 1.0 / 1.0; // Between the encoder and wheel, not between the motor and wheel
```

Comment:
Why is this 1.0/1.0? Are we expecting to change this, if not let's just simplify this to 1.0.

Reply:
I've been writing gear ratios for various subsystems as the product of each of the gearbox stages, so this is just following that convention.

In my opinion, having a ratio written as "60/20 * 16/34 * 45/15" is much easier to think about and maintain on a constantly evolving robot than "17/20" or "4.23529411764...".